src/utilities/validateConnectionArgs.ts

```diff
@@ -0,0 +1,91 @@
+import { GraphQLError } from "graphql";
+
+/**
+ * Checks if a value is not null or undefined.
+ * @param value - The value to check.
+ * @returns True if the value is not null or undefined, otherwise false.
+ */
+export function isNotNullish<T>(value: T | undefined | null): value is T {
+  return value !== undefined && value !== null;
+}
+
+/**
+ * Represents the arguments for Relay-style pagination.
+ */
+export type RelayConnectionArguments = {
+  after?: string | null;
+  before?: string | null;
+  first?: number | null;
+  last?: number | null;
+  limit: number;
```

Comment:
`limit` shouldn't be a part of the relay connection arguments, it is to be provided seperately